presto-docs/src/main/sphinx/functions/binary.rst

```diff
@@ -37,77 +53,76 @@ Binary Functions
     negative starting position is interpreted as being relative to the end of
     the string.
 
-.. function:: to_base64(binary) -> varchar
-
-    Encodes ``binary`` into a base64 string representation.
+Base64 Encoding Functions
+-------------------------
 
 .. function:: from_base64(string) -> varbinary
 
     Decodes binary data from the base64 encoded ``string``.
 
-.. function:: to_base64url(binary) -> varchar
+.. function:: to_base64(binary) -> varchar
 
-    Encodes ``binary`` into a base64 string representation using the URL safe alphabet.
+    Encodes ``binary`` into a base64 string representation.
 
 .. function:: from_base64url(string) -> varbinary
 
     Decodes binary data from the base64 encoded ``string`` using the URL safe alphabet.
 
-.. function:: to_hex(binary) -> varchar
+.. function:: to_base64url(binary) -> varchar
 
-    Encodes ``binary`` into a hex string representation.
+    Encodes ``binary`` into a base64 string representation using the URL safe alphabet.
+
+Hex Encoding Functions
+----------------------
 
 .. function:: from_hex(string) -> varbinary
 
     Decodes binary data from the hex encoded ``string``.
 
-.. function:: to_big_endian_64(bigint) -> varbinary
+.. function:: to_hex(binary) -> varchar
 
-    Encodes ``bigint`` in a 64-bit 2's complement big endian format.
+    Encodes ``binary`` into a hex string representation.
 
-.. function:: from_big_endian_64(binary) -> bigint
+Integer Encoding Functions
+--------------------------
 
-    Decodes ``bigint`` value from a 64-bit 2's complement big endian ``binary``.
+.. function:: from_big_endian_32(binary) -> integer
+
+    Decodes the 32-bit two's complement big-endian ``binary``.
 
 .. function:: to_big_endian_32(integer) -> varbinary
 
-    Encodes ``integer`` in a 32-bit 2's complement big endian format.
+    Encodes ``integer`` into a 32-bit two's complement big-endian format.
 
-.. function:: from_big_endian_32(binary) -> integer
+.. function:: from_big_endian_64(binary) -> bigint
 
-    Decodes ``integer`` value from a 32-bit 2's complement big endian ``binary``.
+    Decodes the 64-bit two's complement big-endian ``binary``.
```

Comment:
These (int, long, float, double) should mention what happens when binary is greater or less than 4 or 8 bytes